apps/web/src/components/TopBar.tsx

```diff
@@ -19,18 +19,19 @@ import { useAuth } from '@web/hooks/useAuth';
 import { PositionsService } from '@web/client';
 import { useQuery } from '@tanstack/react-query';
 
+/**
+ * @returns the top bar of the application
+ */
 export const TopBar: React.FC = () => {
-  const { user, signOut } = useAuth();
-  const { data: positions, error: positionsError } = useQuery({
```

Comment:
nit: why is `positionsError` not needed anymore?

Reply:
It wasn't been used anywhere so I decided to remove it to reduce confusion, we'll add it back if necessary.